pkg/commands/git_commands/commit.go

```diff
@@ -38,6 +38,24 @@ func (self *CommitCommands) SetAuthor(value string) error {
 	return self.cmd.New(cmdArgs).Run()
 }
 
+// Add a commit's coauthor using Github/Gitlab Co-authored-by metadata. Value is expected to be of the form 'Name <Email>'
+func (self *CommitCommands) AddCoAuthor(sha string, value string) error {
+	message, err := self.GetCommitMessage(sha)
+	if err != nil {
+		return err
+	}
+
+	message = message + fmt.Sprintf("\nCo-authored-by: %s", value)
+
+	args := []string{"-m", message}
+	cmdArgs := NewGitCmd("commit").
+		Arg("--allow-empty", "--amend", "--only").
```

Comment:
```suggestion
		Arg("--allow-empty", "--amend", "--only", "-m", message).
```
No need to create a separate `args` variable